datadog-oci-orm/metrics-setup/metrics-function/func.py

```diff
@@ -15,6 +15,9 @@
 
 OUTPUT_MESSAGE_VERSION = "v1.0"
 
+# Switch value to True to get payload logs 
+DETAILED_LOGGING_ENABLED = False
```

Comment:
Can we add this as an environment var, similar to how we do for `DD_INTAKE_HOST` and `DD_API_KEY`? That way turning this off/on doesn't require a new function image/deployment.